provider/azure/networking.go

```diff
@@ -73,37 +73,41 @@ const (
 	securityRuleInternalAPIInbound
 )
 
-var (
-	sshSecurityRule = network.SecurityRule{
-		Name: to.StringPtr("SSHInbound"),
-		SecurityRulePropertiesFormat: &network.SecurityRulePropertiesFormat{
-			Description:              to.StringPtr("Allow SSH access to all machines"),
-			Protocol:                 network.SecurityRuleProtocolTCP,
-			SourceAddressPrefix:      to.StringPtr("*"),
-			SourcePortRange:          to.StringPtr("*"),
-			DestinationAddressPrefix: to.StringPtr("*"),
-			DestinationPortRange:     to.StringPtr("22"),
-			Access:                   network.SecurityRuleAccessAllow,
-			Priority:                 to.Int32Ptr(securityRuleInternalSSHInbound),
-			Direction:                network.SecurityRuleDirectionInbound,
-		},
-	}
+var sshSecurityRule = network.SecurityRule{
```

Comment:
Can't we use newSecurityRule here (slightly modified, I guess to permit specification of priority) ?

Reply:
Will try.

Reply:
Done.